### PR TITLE
[Internal release 03 07 23] - Literals changes are done along with other missing texts in locals.

### PR DESCRIPTION
**Pull request by Anil-kumar-Majji**

### CODE REVIEW

File: locales/fil.json [hunk 1]
```diff
@@ -112,8 +143,8 @@
     "aboutDetailes": "Ang Inji ay isang mobile app na maaaring gamitin bilang digital wallet para mag-imbak ng mga kredensyal. Pinapayagan din nito ang pag-verify ng pagkakakilanlan sa parehong offline at online na mga mode, sa anumang lugar at oras.",
     "forMoreDetailes": "Para sa karagdagang detalye",
     "clickHere": "Pindutin dito",
-    "version": "Bersyon:",
-    "tuvaliVersion": "Tuvali-bersyon"
+    "version": "Bersyon: 0.4.1",
```
Comment (vijay151096) [inline locales/fil.json:146]: Version should not be hardcode, it should be extracted from package.json

Comment (vijay151096) [reply locales/fil.json:146]: please do check for other languages files as well

Comment (Anil-kumar-Majji) [reply locales/fil.json]: Removed version. tuvali-version in all the locals of AboutInji obj. They are further no need.

**Review by vijay151096 — APPROVED** [review]
looks good to me.

**Pull request merged**